cli/download_source_test.go

```diff
@@ -21,6 +21,24 @@ import (
 	"github.com/gruntwork-io/terragrunt/util"
 )
 
+func TestAlreadyHaveLatestCodeLocalFilePathWithHash(t *testing.T) {
+	t.Parallel()
+
+	canonicalUrl := fmt.Sprintf("file://%s", absPath(t, "../test/fixture-download-source/hello-world-local"))
+	downloadDir := fmt.Sprintf("%s", absPath(t, "./test-fixtures/download-dir-local-no-changes"))
```

Comment:
Any reason we can't make this download dir a tmpDir? Does it need to persist in the codebase?